core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListHelper.java

```diff
@@ -90,6 +105,62 @@ private ListHelper() {
             .collect(Collectors.toList());
     }
 
+    public static Page createList(ResourceResolver resourceResolver, String parentPath, String pageName, List<SimpleListItem> values)
+        throws WCMException, PersistenceException {
```

Comment:
perhaps we can handle these exceptions ourselves? I don't think that the user can do with this exception other than just log it)
@smiakchilo , @Alexey-Chernyakov , what do you think?

Reply:
I think it's better to log it and maybe to have our exception here

Reply:
We definitely need an exception thrown. Otherwise, a user will have his `page == null` and no idea why it is null.
Not sure we need an exception of our own here. Whet will it change& A user will anyway need to try-catch. 
I've looked through ACS Commons. Some of the utility methods do throw exceptions 
![image](https://user-images.githubusercontent.com/48795460/142434263-ba88b36c-3faf-4e3b-bcee-dc37a3439141.png)
.. while others don't
Are there really strong arguments for a custom exception? Or we can do with the standard ones?

Reply:
I suggest using the standard